.github/workflows/build-test-publish-apps.yml

```diff
@@ -14,7 +14,10 @@ on:
 
 env:
   CUT_RELEASE_PR: ${{ github.event_name == 'pull_request' && (contains(github.event.pull_request.title, 'Cut release v')) }}
-  BUILD_RELEASE: ${{ github.event_name == 'release' || github.event_name == 'schedule' || github.event_name == 'pull_request' && (contains(github.event.pull_request.title, 'Cut release v')) }}
+  # NIGHTLY: ${{ github.event_name == 'schedule' }}
+  NIGHTLY: true
+  # BUILD_RELEASE: ${{ github.event_name == 'release' || github.event_name == 'schedule' || github.event_name == 'pull_request' && (contains(github.event.pull_request.title, 'Cut release v')) }}
+  BUILD_RELEASE: true
```

Comment:
Will flip back to the boolean evals before merging. This new NIGHTLY var will help flipping nightly tests on moving forward on branches.